src/client/index.ts

```diff
@@ -143,7 +143,7 @@ type RunMutationCtx = {
   runMutation: GenericMutationCtx<GenericDataModel>["runMutation"];
 };
 
-export type OpaqueIds<T> = T extends GenericId<infer _T> | string
+export type OpaqueIds<T> = T extends GenericId<infer _T>
```

Comment:
btw this is a really important part of the change https://github.com/get-convex/templates/pull/38